app/src/main/java/protect/card_locker/DBHelper.java

```diff
@@ -189,6 +193,7 @@ public void onUpgrade(SQLiteDatabase db, int oldVersion, int newVersion) {
                     LoyaltyCardDbIds.HEADER_COLOR + " INTEGER," +
                     LoyaltyCardDbIds.CARD_ID + " TEXT not null," +
                     LoyaltyCardDbIds.BARCODE_ID + " TEXT," +
+                    LoyaltyCardDbIds.USAGE_NUMBER + " INTEGER DEFAULT '0', " +
```

Comment:
This is wrong, this will only be executed when users are on database version 9. So existing installations won't get the usage column. This should be removed (there should be an extra upgrade at the end and the DATABASE_VERSION variable needs to be increased).

Reply:
in progress..